haiku-misc/space/space-1.2.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="A space screensaver"
+DESCRIPTION="Space uses real math to give users an awesome, \
+smooth, stellar experience, in real perspective."
+HOMEPAGE="https://github.com/HaikuArchives/Space"
+COPYRIGHT="David Enderso"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="80216ae2e0821d122f447a1671d790e41094ecc7"
+SOURCE_URI="https://github.com/HaikuArchives/space/archive/$srcGitRev.tar.gz"
```

Comment:
/space/ :arrow_right: /**S**pace/
```diff
-SOURCE_URI="https://github.com/HaikuArchives/space/archive/$srcGitRev.tar.gz"
+SOURCE_URI="https://github.com/HaikuArchives/Space/archive/$srcGitRev.tar.gz"
```
(This is not required but since the project was registered with a capital S it will look nicer and more consistent with the capital S we already have in `HOMEPAGE`.)